src/expand.rs

```diff
@@ -128,29 +128,22 @@ where
 
     let mut failures = 0;
     for test in tests {
-        let file_stem = test
-            .test
-            .file_stem()
-            .expect("no file stem")
-            .to_string_lossy()
-            .into_owned();
+        let path = test.test.display();
+        let expanded_path = test.test.with_extension("expanded.rs");
 
         match test.run(&project, expansion_behavior, &args) {
             Ok(outcome) => match outcome {
-                ExpansionOutcome::Same => println!("{} - ok", file_stem),
+                ExpansionOutcome::Same => {
+                    let _ = writeln!(std::io::stdout(), "{} - ok", path);
+                }
```

Comment:
This change isn't directly related to this fix, but `cargo test` doesn't print the output from `println!`, so this is needed to make sure the path was output correctly.

Reply:
Right. It prints it only when something fails, for example, when one of the expansions failed, you can see which ones were ok.